### DISCUSSION TITLE
I don't find any way about how to create a user.

### DISCUSSION THREAD

**Discussion by zszen** [gocd/gocd#12643] [Q&A] [answered]
How can I login without anonymous ?

**Comment by chadlwilson** [comment, marked as answer]
https://docs.gocd.org/current/configuration/dev_authentication.html
https://www.gocd.org/plugins/#authorization

**Comment by zszen** [comment]
> https://docs.gocd.org/current/configuration/dev_authentication.html

Use password method. I can create a user for login. But I add user into it. the user list page don't update.

**Reply by zszen** [reply]
I got it. Import user.

**Reply by chadlwilson** [reply]
If I recall correctly you can either import user or the user entries will be created 'on-demand' when the person logs in the first time.
